docs/topics/jvm/jvm-records.md

```diff
@@ -0,0 +1,74 @@
+[//]: # (title: Records)
+
+> JVM records feature is [Experimental](components-stability.md). It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-42430).
+>
+{type="warning"}
+
+_Records_ are Java specific classes for storing immutable data. Records carry a fixed set of values – the _records components_.
```

Comment:
Do you mean `Java-specific classes` or `specific Java classes`? It not clear for me now, maybe it's better to rephrase.
I'd also add a link to the Java doc page about records.

Reply:
Rephrased to be more precise.